lib/galaxy/workflow/modules.py

```diff
@@ -81,6 +81,9 @@ def get_type( self ):
     def get_name( self ):
         return self.name
 
+    def get_version( self ):
+        return None
```

Comment:
Thanks for the quick review. Its so much appreciated. These placeholders are for the backend. The front-end does not change if a response attribute is set to `None` or not available at all. The alternative here would be to check if/else a module has `get_version()` or `get_post_(job_)actions()` before calling those instead of calling these placeholders. In the future I would like to rename `get_post_job_actions` to `get_post_actions`. My suggestion here is that its cleaner and more readible to use these placeholders instead of distributing if/else-statements through the code-base. It also comes with the benefit that these concepts are available for other modules if needed e.g. we could version subworkflows by the last date they were modified and then offer to update them when a workflow is loaded into the editor. Another application could be related to certain non-tool modules once their descriptions are moved to external yaml/xml-files.